activestorage/lib/active_storage/errors.rb

```diff
@@ -1,11 +1,14 @@
 # frozen_string_literal: true
 
 module ActiveStorage
-  class InvariableError < StandardError; end
-  class UnpreviewableError < StandardError; end
-  class UnrepresentableError < StandardError; end
+  # Generic Active Storage exception class.
+  class ActiveStorageError < StandardError; end
```

Comment:
I realize Active Record has `ActiveRecord::ActiveRecordError` and Action Controller has `ActionController::ActionControllerError`, but I don’t think the redundancy is necessary here. Let’s go with `ActiveStorage::Error`.